src/p2p/net_node.inl

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2014-2019, The Monero Project
-// Copyright (c)      2018, The Loki Project
```

Comment:
Because I have looked into copyright quite extensively, the full answer is:

The copyright year is actually pretty pointless legally: copyright applies from the date something was created, regardless of what year is claimed.  A new file published in 2019 is always copyright 2019 regardless of whether *parts* of the file were published earlier, as would be the case here.  The only thing that would technically matter here would licence compliant, but since we are the copyright holders we aren't actually subject to that license and so can theoretically change it (though cannot change Monero's statement).  Putting "2018-2019" only serves as a reminder to someone who might be looking that there are some portions for which a copyright beginning in 2018 apply.  I don't see much value in pointing that out, though: all that does is remind people that some of the code has a copyright that ends earlier (e.g. so they can use some code outside the license in 2068 instead of 2069).

But the short answer is that this was just an oversight.